pkg/sdk/go/domains.go

```diff
@@ -56,6 +57,9 @@ func (sdk mgSDK) UpdateDomain(domain Domain, token string) (Domain, errors.SDKEr
 		return Domain{}, errors.NewSDKError(err)
 	}
 
+	if domain.ID == "" {
+		return Domain{}, errors.NewSDKError(apiutil.ErrMissingID)
+	}
```

Comment:
Is there any other way instead of using apiutils?

Reply:
I can define a new sdk error with status if needed.

Reply:
Lets go with now, later we can move errors if needed.